packages/shared/src/consts.ts

```diff
@@ -3,7 +3,10 @@ import { ChainflipNetwork, ChainflipNetworks } from './enums';
 const POOLS_NETWORK_FEE_HUNDREDTH_PIPS: Partial<
   Record<ChainflipNetwork, number>
 > = {
-  [ChainflipNetworks.perseverance]: 100,
+  [ChainflipNetworks.backspin]: 1000,
```

Comment:
unfortunately this is not available on any rpc yet

Reply:
https://linear.app/chainflip/issue/WEB-683/use-network-fee-from-rpc-instead-of-hardcoding-it